src/libsyntax/parse/lexer/mod.rs

```diff
@@ -1911,7 +1911,7 @@ mod tests {
 
     use crate::ast::{Ident, CrateConfig};
     use crate::symbol::Symbol;
-    use crate::source_map::SourceMap;
+    use crate::source_map::{SourceMap, FilePathMapping};
```

Comment:
This seems included by mistake.

Reply:
:+1: that was indeed intended for https://github.com/rust-lang/rust/pull/59687